src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -119,13 +137,18 @@ class Comment extends React.Component<Props, State> {
             getMentionWithQuery,
             mentionSelectorContacts,
             onEdit,
+            parent,
+            status,
         } = this.props;
         const { isConfirmingDelete, isEditing, isInputOpen } = this.state;
         const createdAtTimestamp = new Date(created_at).getTime();
         const createdByUser = created_by || PLACEHOLDER_USER;
         const canEdit = onEdit !== noop && permissions.can_edit;
         const canDelete = permissions.can_delete;
-        const isMenuVisible = (canDelete || canEdit) && !isPending;
+        // Replies can't be resolved
+        const canResolve = !parent && onEdit !== noop && permissions.can_resolve;
```

Comment:
Is the `!parent` check needed here? Does `permissions.can_resolve` check whether the comment is a reply? If not, should it?